servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/ProtocolCompatibilityTest.java

```diff
@@ -610,6 +611,50 @@ void clientH2ReturnStatus() throws Exception {
         }
     }
 
+    @Test
+    void grpcJavaToServiceTalkUnimplementedService() throws Exception {
+        try (TestServerContext server = serviceTalkServerBlocking(ErrorMode.STATUS, false, null, null);
+             CompatClient client = grpcJavaClient(server.listenAddress(), null, false, null)) {
+            final Single<CompatResponse> response =
+                    client.unimplementedServerCall(CompatRequest.newBuilder().setId(1).build());
+            validateGrpcErrorInResponse(response.toFuture(), false, UNIMPLEMENTED,
+                    "Method grpc.netty.Compat/unimplementedServerCall is unimplemented");
+        }
+    }
+
+    @Test
+    void grpcJavaToGrpcJavaUnimplementedService() throws Exception {
+        try (TestServerContext server = grpcJavaServer(ErrorMode.NONE, false, null, null);
+             CompatClient client = grpcJavaClient(server.listenAddress(), null, false, null)) {
+            final Single<CompatResponse> response =
+                    client.unimplementedServerCall(CompatRequest.newBuilder().setId(1).build());
+            validateGrpcErrorInResponse(response.toFuture(), false, UNIMPLEMENTED,
+                    "Method grpc.netty.Compat/unimplementedServerCall is unimplemented");
+        }
+    }
+
+    @Test
+    void serviceTalkToGrpcJavaUnimplementedService() throws Exception {
+        try (TestServerContext server = grpcJavaServer(ErrorMode.NONE, false, null, null);
+             CompatClient client = serviceTalkClient(server.listenAddress(), false, null, null)) {
+            final Single<CompatResponse> response =
+                    client.unimplementedServerCall(CompatRequest.newBuilder().setId(1).build());
+            validateGrpcErrorInResponse(response.toFuture(), false, UNIMPLEMENTED,
+                    "Method grpc.netty.Compat/unimplementedServerCall is unimplemented");
+        }
+    }
+
+    @Test
+    void serviceTalkToServiceTalkUnimplementedService() throws Exception {
+        try (TestServerContext server = serviceTalkServerBlocking(ErrorMode.NONE, false, null, null);
```

Comment:
For `grpcJavaToServiceTalkUnimplementedService` and `serviceTalkToServiceTalkUnimplementedService` consider making them `@ParameterizedTest` to also run with `serviceTalkServer(..., new CompatService() { })`